tensor_flow_block.py

```diff
@@ -76,7 +75,27 @@ class NetworkConfig(PropertyHolder):
                                default=Optimizers.GradientDescentOptimizer)
     dropout = FloatProperty(title='Dropout Percentage During Training',
                             default=0)
-    random_seed = IntProperty(title="Random seed", default=0, visible=False)
+    random_seed = IntProperty(title="Random Seed", default=0, visible=False)
+
+
+class ModelManagement(PropertyHolder):
+
+    default_tag = '{{ datetime.datetime.now().strftime(\'%Y%m%d%H%M%S\') }}'
+    save_file = StringProperty(title='Save Weights to File',
+                               default='',
+                               allow_none=True)
+    load_file = StringProperty(title='Load Weights From File',
+                               default='',
+                               allow_none=True)
+    tensorboard_int = IntProperty(title='Train Steps per TensorBoard Record',
+                                  default=0)
+    tensorboard_tag = StringProperty(title='TensorBoard Run Label',
+                                     default=default_tag,
+                                     allow_none=True,
+                                     visible=False)
+    tensorboard_dir = StringProperty(title='TensorBoard File Directory',
```

Comment:
Why not just always throw the tf logs into the project directory?  That way we guarantee that the user will have read/write permission to that directory.

Reply:
that's the default behavior, `<project-root>/tensorboard/`. users that need to save to e.g a network drive can change hidden prop to whatever is needed.

Reply:
Works for me